crates/bevy_ecs/src/world/unsafe_world_cell.rs

```diff
@@ -996,6 +996,16 @@ impl<'w> UnsafeEntityCell<'w> {
             })
         }
     }
+
+    /// Returns the source code location from which this entity has last been spawned
+    /// or despawned.
+    #[cfg(feature = "track_change_detection")]
+    pub fn get_spawned_despawned_by(self) -> &'static Location<'static> {
+        self.world()
+            .entities()
+            .get_entity_spawned_despawned_by(self.entity)
+            .unwrap()
+    }
```

Comment:
I suspect this documentation is what should be used for the above methods instead.

Reply:
The `get_` can also be dropped